docs/installation/local.md

```diff
@@ -49,6 +49,12 @@ sudo apt-get install -y mongodb
 sudo service mongod start
 ```
 
+if `sudo service mongod start` throws "Failed to start mongod.service" error then execute
```

Comment:
Simplify to: if the final command raises a "__Failed to start mongod.service__" error then execute `sudo service mongodb start`